scripts/corelli/calibration/bank.py

```diff
@@ -338,16 +418,21 @@ def calibrate_bank(workspace: WorkspaceTypes,
     table of calibrated detector IDs and a table of non-calibrated detector IDs
 
     This function produces the following temporary workspaces: 'PeakTable' (TableWorkspace),
-    'parameters_table' (WorkspaceGroup), and 'acceptance' (Workspace2D). Thus, any extant workspace having
-    any of these names will be overwritten
+    'PeakTable' (TableWorkspace), 'ParametersTable' (WorkspaceGroup), and 'acceptance' (Workspace2D).
```

Comment:
I believe we switched from acceptance to "fit"

Reply:
In this context, the `acceptance` table is appropriate. The `fit` table is a merge of the `acceptance` table and the `parameters` table.
The `acceptance` table contains information about the merit of each tube-calibration. The `parameters` table lists some optimized fitting parameters for each tube-calibration. Feng wanted to save both pieces of info so they all go into the `fit` table.

Reply:
Got it, thanks!